content/docs/enterprise/changelog.mdx

```diff
@@ -15,6 +15,34 @@ import Versioning from '../admonitions/_semantic-versioning.mdx';
 
 :::
 
+## 0.27.0
+
+### Improvements
+
+- The Enterprise Console now includes a “Report issue” feedback widget, allowing you to easily report any problems you may encounter. We look forward to hearing your feedback!
+
+  This widget is loaded from a third-party service. If you do not wish to include this on your Enterprise Console installation, you can disable the feedback widget by setting the new option `--disable-feedback-widget`.
+
+- IdP directory sync performance has been improved, especially when using Okta.
+- The policy builder has a new “Exists” condition for use with external data source records. This condition is true when an incoming request matches any record in the selected external data source.
+
+  As an example use case: you could maintain an external data source with a list of approved client certificates (keyed by certificate fingerprint), and enforce that incoming requests must match one of the approved client certificates.
+
+- When configuring an external data source, the ”Foreign Key” input will now display all valid choices, making it easier to configure.
+- The `--disable-validation` option has been expanded to include an additional validation mode. The existing validation modes are now represented by a new option `--validation-mode`. There are now three modes:
+
+  - `full`, the default validation mode
+  - `static`, a lighter-weight validation mode that should still catch most potential issues
+  - `none`, equivalent to the existing `--disable-validation` option
+
+  Additionally, the `none` mode now also disables a safety check related to overlapping certificate domain names.
+
+### Bug fixes
```

Comment:
Other versions use "Fixed" for this section.

Reply:
Updated.